From 74b7c656b0a5325c8198ee137bac05878c99d585 Mon Sep 17 00:00:00 2001
From: Alon Bar-Lev <alon.barlev@gmail.com>
Date: Sat, 29 Oct 2016 11:39:29 +0300
Subject: [PATCH] build: fix symlink installation

this fixes incorrect fix b313f00cc of symlink installation
by adding dependency instead of overwriting automake behavior.

Signed-off-by: Alon Bar-Lev <alon.barlev@gmail.com>
---
 src/Makefile.am | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/src/Makefile.am b/src/Makefile.am
index 2aa5195..d0dd37e 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -48,7 +48,7 @@ pkcs11_la_LDFLAGS = $(AM_LDFLAGS) -module -shared -shrext $(SHARED_EXT) \
 check-local: $(LTLIBRARIES)
 	cd .libs && $(LN_S) -f pkcs11$(SHARED_EXT) libpkcs11$(SHARED_EXT)
-install-exec-hook:
+install-exec-hook: install-enginesLTLIBRARIES
 	cd '$(DESTDIR)$(enginesdir)' && $(LN_S) -f pkcs11$(SHARED_EXT) libpkcs11$(SHARED_EXT)
 
 if WIN32
 # def file required for MS users to build library
-- 
2.7.3

